shifthelper/checks.py

```diff
@@ -1,53 +1,84 @@
 from custos import IntervalCheck
-import smart_fact_crawler
+import smart_fact_crawler as sfc
 from .tools.is_shift import is_shift_at_the_moment
+import requests
+from abc import ABCMeta, abstractmethod
```

Comment:
Why not using a base class until the very moment when 'NotImplementedError' is risen? The more restrictive ABC concept is not pythonic in my opinion. Further, it is one import more and might be less common than the std exception 'NotImplementedError'. I vote for rising 'NotImplementedError'.